packages/messaging/README.md

```diff
@@ -17,7 +17,7 @@ The messaging extension has the following peer dependenices, which must be insta
 
 See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native#requirements) instructions on the main page
 
-**Installation instructions for `@adobe/react-native-aepmessaging` v1.0.0-beta.3 with in-app messaging support**   
+**Installation instructions for `@adobe/react-native-aepmessaging` v1.0.0-beta.3 with in-app messaging support**  
```

Comment:
We will need find out if this instruction still valid.  And it won't be Installation instructions for `@adobe/react-native-aepmessaging` v1.0.0-beta.3 anymore since the version will be changed.

Reply:
We have discussed it offline, the fix will be in another PR.